src/units/attack_type.cpp

```diff
@@ -288,6 +300,7 @@ bool attack_type::apply_modification(const config& cfg)
 	const std::string& set_icon = cfg["set_icon"];
 	const std::string& del_specials = cfg["remove_specials"];
 	auto set_specials = cfg.optional_child("set_specials");
+	auto remove_specials = cfg.optional_child("remove_specials");
```

Comment:
Well, this is interesting. I think this would be the first time we support a key and a tag with the same name in a given context where it's valid to have _both_. (We do support either key or tag for an AI aspect's `value`, but there you can only choose one.)

It's not a problem per se, but is it something we'd prefer to avoid? @stevecotton What do you think?

Reply:
Could you suggest another name ,then?

Reply:
I'm tempted to say it's okay like this, but I'm calling it to attention because it's unusual and I think I should make sure no-one else has a problem with it.

Reply:
removes_specials changed to removes_specials_filter